Base estructure of the frontend

YxMARCKxY opened the pull request:
- We transferred the Figma design of the employee check in/out screen to HTML and CSS
- We use Javascript to create the live clock
	- The function 'startTime()' and 'checkTime()' were created to inject the time
	  obtained from the system through the Date() class in the HTML

# Description

We added the first changes/estructure according to the frontend mockups

diegoOG09 committed:
Base estructure of the frontend

- We transferred the Figma design of the employee check in/out screen to HTML and CSS
- We use Javascript to create the live clock
	- The function 'startTime()' and 'checkTime()' were created to inject the time
	  obtained from the system through the Date() class in the HTML

Co-authored-by: YxMARCKxY <mramirez73@ucol.mx>
Co-authored-by: armanirmr <armanirmr@gmail.com>
Co-authored-by: rene3255 <ranchoeltecal@gmail.com>

JuanequeX commented on css/styles.css:
```diff
@@ -0,0 +1,267 @@
+  box-shadow: rgba(0, 0, 0, 0.12) 0px 1px 3px, rgba(0, 0, 0, 0.24) 0px 1px 2px;
+}
+
+/* .content-nogrid {
+  display: grid;
+  grid-template-columns: 25% 71%;
+  margin-top: 7rem;
+} */
```
If this code are not necessary, can you remove it, what do you think?

YilKanda replied:
Totally agree with @JuanequeX

JuanequeX commented on css/styles.css:
```diff
@@ -0,0 +1,267 @@
+select > option {
+background-color: white;
+color: #2c3841;
+}
```
Add a blank line here

JuanequeX commented on index.html:
```diff
@@ -0,0 +1,83 @@
+
+</body>
+
+</html>
```
Add blank line here

JuanequeX reviewed (changes requested):
It is good practice to order the lines of code in your style blocks in alphabetical order, I recommend that you sort it.

in visual studio code can use `ctrl + shift + p` and select the option `sort lines ascending` for to do

JuanequeX commented:
Just add the co-authors into the commit, you don't need to put it in your pull request description

YilKanda commented on js/timer-script.js:
```diff
@@ -0,0 +1,35 @@
+  document.getElementById('h').innerHTML = h + ":";
+  document.getElementById('m').innerHTML = m + ":";
+  document.getElementById('s').innerHTML = s;
+  //document.getElementById('txt').innerHTML =  h + ":" + m + ":" + s;
```
If you don't occupy this line, delete it.

YilKanda commented on js/timer-script.js:
```diff
@@ -0,0 +1,35 @@
+function checkTime(i) {
+  if (i < 10) { i = "0" + i };  // add zero in front of numbers < 10
+  return i;
+}
```
Add a blank line.

YilKanda reviewed (commented):
I would recommend focusing on the backend part first and then the frontend part. 

It is not bad that you use a template, but it is important that you know what that template does and see if you can do something similar with a lesser amount of code, that is, optimize it.

Finally, I would like to add a phrase from Addy Osmani:

"First do it, then do it right, then do it better."

As a tip, first carry out the functionality of the project (backend), and then focus on making it look nice (frontend).

diegoOG09 committed:
'admin', 'employees' and 'branches' views done

- 'admin screen': it contains a admin icon, username and password input and login button.
				  A new link was added in the navigation bar to generate reports

- 'employee screen': it contains a employees table and a button to add employee
- 'branches screen': it contains a branches table and a button to add branch

- In all of this screens was added arrows for navigation between employees and branches tables

Co-authored-by: armanirmr <armanirmr@gmail.com>
Co-authored-by: YxMARCKxY <mramirez73@ucol.mx>

diegoOG09 committed:
Views added and corrected the details mentioned in the code review

- 'admin' view added
- 'branches' view added
- 'employees' view added

Co-authored-by: armanirmr <armanirmr@gmail.com>
Co-authored-by: YxMARCKxY <mramirez73@ucol.mx>
Co-authored-by: rene3255 <ranchoeltecal@gmail.com>

YilKanda commented on css/styles.css:
```diff
@@ -0,0 +1,303 @@
+  font-size: 3rem;
+}
+
+/*#arrow_left{
+ 
+}
+
+#arrow_right{
+  
+}*/
```
If you don't occupy this code, delete it.

YilKanda commented on views/employees.html:
```diff
@@ -0,0 +1,113 @@
+    </div>
+    
+  </div>
+
+
+
+
```
Remove unnecessary spaces,

YilKanda commented on views/employees.html:
```diff
@@ -0,0 +1,113 @@
+
+</body>
+
+</html>
```
At the end of each file, leave a blank line.

YilKanda reviewed (commented):
Check all your files, in which you have unnecessary blank spaces.

Al the end of each file, leave a blank line.

You can check your * .html files with [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).

diegoOG09 committed:
Corrections made

Co-authored-by: armanirmr <armanirmr@gmail.com>
Co-authored-by: YxMARCKxY <mramirez73@ucol.mx>
Co-authored-by: rene3255 <ranchoeltecal@gmail.com>

JuanequeX reviewed (approved):
LGTM
